app/src/behaviors/behavior_sensor_rotate.c

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright (c) 2022 The ZMK Contributors
+ *
+ * SPDX-License-Identifier: MIT
+ */
+
+#define DT_DRV_COMPAT zmk_behavior_sensor_rotate
+
+#include <drivers/behavior.h>
+
+#include <zephyr/device.h>
+#include <zephyr/drivers/sensor.h>
+#include <zephyr/logging/log.h>
+
+#include <zmk/event_manager.h>
+#include <zmk/events/keycode_state_changed.h>
+#include <zmk/behavior_queue.h>
+#include <zmk/keymap.h>
+
+LOG_MODULE_DECLARE(zmk, CONFIG_ZMK_LOG_LEVEL);
+
+struct behavior_sensor_rotate_config {
+    struct zmk_behavior_binding cw_binding;
+    struct zmk_behavior_binding ccw_binding;
+    int tap_ms;
+};
+
+static int on_sensor_binding_triggered(struct zmk_behavior_binding *binding,
+                                       const struct device *sensor,
+                                       struct zmk_behavior_binding_event event) {
+    const struct device *dev = device_get_binding(binding->behavior_dev);
+    const struct behavior_sensor_rotate_config *cfg = dev->config;
+
+    struct sensor_value value;
+    int err;
+
+    err = sensor_channel_get(sensor, SENSOR_CHAN_ROTATION, &value);
```

Comment:
Minor: 

```suggestion
    int err = sensor_channel_get(sensor, SENSOR_CHAN_ROTATION, &value);
```

or

```suggestion
    const int err = sensor_channel_get(sensor, SENSOR_CHAN_ROTATION, &value);
```
since it isn't ever changed.